DataScience/LogDownloader.py

```diff
@@ -69,7 +69,7 @@ def add_parser_args(parser):
 def update_progress(current, total):
     barLength = 50 # Length of the progress bar
     progress = current/total
-    if int(progress*1000)%10 == 0: 
+    if int(progress*1000)%10 == 0 or '__name__' == '__main__': 
```

Comment:
This shouldn't be the place where we address if writing or not to. update_progress() shouldn't be called if no display is needed

Reply:
Added a default behavior to report progress for LogDownloader